crates/forge/src/scarb.rs

```diff
@@ -179,12 +179,21 @@ pub fn paths_for_package(
         .ok_or_else(|| anyhow!("Failed to find metadata for package = {package}"))?;
 
     let package_path = package.root.clone();
-
     let lib_path = compilation_unit.target.source_path.clone();
 
     Ok((package_path, lib_path))
 }
 
+pub fn target_dir_for_package(package_path: &Utf8PathBuf) -> Result<Utf8PathBuf> {
+    let scarb_metadata = MetadataCommand::new().inherit_stderr().exec()?;
+
+    let target_dir = scarb_metadata
+        .target_dir
+        .unwrap_or_else(|| package_path.join("target/dev"));
```

Comment:
We also join `dev` in `try_get_starknet_artifacts_path`. Won't this cause problems?

Reply:
My mistake here